db_stress_tool/expected_value.h

```diff
@@ -181,9 +230,12 @@ class PendingExpectedValue {
   uint32_t GetFinalValueBase() { return final_value_.GetValueBase(); }
 
  private:
-  std::atomic<uint32_t>* const value_ptr_;
-  const ExpectedValue orig_value_;
-  const ExpectedValue final_value_;
+  inline void ClosePendingState() const { pending_state_closed_ = true; }
+
+  std::atomic<uint32_t>* value_ptr_;
+  ExpectedValue orig_value_;
+  ExpectedValue final_value_;
+  mutable bool pending_state_closed_ = false;
```

Comment:
nit: initialize this in constructor so not to mix inline and constructor initialization